staging/src/k8s.io/kubectl/pkg/cmd/create/create_priorityclass.go

```diff
@@ -17,13 +17,21 @@ limitations under the License.
 package create
 
 import (
+	"context"
+	"fmt"
+
 	"github.com/spf13/cobra"
 
-	apiv1 "k8s.io/api/core/v1"
+	corev1 "k8s.io/api/core/v1"
+	scheduling "k8s.io/api/scheduling/v1"
```

Comment:
`schedulingv1` please to keep convention :wink:

Reply:
done 🙂